index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/main.js" defer="defer">
+
+    </script>
+    <title> Calculator </title>
+</head>
+
+<body>
+    <input id="screen" type="text">
```

Comment:
add class

Reply:
use `type="number"` for numbers

Reply:
I tried this one, but unfortunately it is not applicable here. Firstly, if I divide 6 by  0 it won't be able to show "Infinity" result. Secondly, when I add "." it also refuses to write it in the input.